larky/src/main/resources/vgs/nts.star

```diff
@@ -50,12 +54,14 @@ def render(
     """
     pan_value = jsonpath_ng.parse(pan).find(input).value
     cvv_value = jsonpath_ng.parse(cvv).find(input).value
+    dcvv_value = jsonpath_ng.parse(dcvv).find(input).value

```

Comment:
this value suppose to be optional? I guess `cvv` and `dvcc`, in real life we probably is going to use one of them but not both of them, right? @mjallday

if so, we may need to use only one of them here?

Reply:
yes, it's an either / or. no _and_